valhalla/baldr/graphreader.h

```diff
@@ -922,6 +922,7 @@ class GraphReader {
     std::unordered_map<uint64_t, std::pair<char*, size_t>> traffic_tiles;
     std::shared_ptr<midgard::tar> archive;
     std::shared_ptr<midgard::tar> traffic_archive;
+    bool traffic_from_index = false;
```

Comment:
to tell the lambda whether it should insert tile indices into `tiles ` or `traffic_tiles`